src/Components/FlowEntry.js

```diff
@@ -1,14 +1,34 @@
 import React from "react";
 // flow rate and pump time entries for a grow bed (water pump page)
 import TextField from '@mui/material/TextField';
+import { db } from '../firebase';
+import { doc, updateDoc } from "firebase/firestore";
+
+const FlowEntry = ({bed}/*{flow, time, handleFlowChange, handleTimeChange}*/) => {
+    const [flow, setFlow] = React.useState('');
```

Comment:
Can we use a React effect to grab the initial states of these variable from the Firebase?